cmake/multi_image.cmake

```diff
@@ -234,10 +234,9 @@ function(add_child_image_from_source)
   elseif (NOT ACI_BOARD)
     # No BOARD is given as argument, this triggers automatic conversion of
     # *.ns board from parent image.
+    get_board_without_ns_suffix(${BOARD}${BOARD_QUALIFIERS} ACI_BOARD)
```

Comment:
something does not sound correct.
In old HWM then the `BOARD` should not contain the revision at this point in time.

In ncs 2.6.x the revision was already stripped from the `BOARD` variable at this point in time.
This could be a regression in support for old HWM, so taking an extra look.

Blocking the PR in the meantime.

Reply:
No changes needed upstgream.

All that need to be changes is to remove all board revision specific handling, see other suggestion fix.

Board revision is passed independently to the child image here:
https://github.com/nrfconnect/sdk-nrf/blob/e6e4696ce35340b475b402dc84a2852d3a960b3a/cmake/multi_image.cmake#L338-L339

Reply:
I think it makes sense. Thanks for investigating.